app/components/access_panels/library_location_popover_component.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module AccessPanels
+  class LibraryLocationPopoverComponent < ViewComponent::Base
+    # location.mhld
+    def initialize(mhld)
+      @mhld = mhld
+      super
+    end
+
+    def render?
+      @mhld.present? && @mhld.any? { |mhld_item| mhld_item.library_has.present? }
```

Comment:
```suggestion
      @mhld.any? { |mhld_item| mhld_item.library_has.present? }
```

Do we need to check for `@mhdl`?  Doesn't Library#mhld aways return an array?

Reply:
Appears so. Changed.